addons/website_slides/views/rating_rating_views.xml

```diff
@@ -109,21 +125,86 @@
         <field name="view_id" ref="rating_rating_view_kanban_slide_channel"/>
         <field name="help" type="html">
             <p class="o_view_nocontent_empty_folder">
-                There are no ratings for these courses at the moment
+                There are no reviews for these courses at the moment
             </p>
         </field>
     </record>
 
+
+    <record id="rating_rating_view_form" model="ir.ui.view">
```

Comment:
Is'it better to make the view from scratch ? (because you remove more field than fields which should be in the view)

https://www.awesomescreenshot.com/image/10588252?key=92c9be9f636ef2e18325b4dba3a7c2a2